src/webviews/main.ts

```diff
@@ -112,6 +112,10 @@ export async function createVueWebview<TRequest, TResponse>(params: WebviewParam
 </html>`
 
     if (params.initialCalls) {
+        // give the webview a little more time before adding content
```

Comment:
Consider marking it a TODO somehow so that it is clear this should be revisited
```suggestion
        // TODO: workaround for cloud9 webviews, remove after cloud9 issue is fixed.
```